### PR TITLE
feat(suite-native): discovery account labels

### LINKED ISSUE #9501: Discovery 6. - imported account names
When there are any accounts found during the discovery, they do not have `accountLabel` set. This behaviour is based on desktop suite where the account labeling is disabled by default. 

Since we have different approach to account labels in mobile suite so far. We need to decide how should be accounts labeled after discovery import.

### PR DESCRIPTION
**Pull request by PeKne**
Mobile app discovered accounts are labeled by name of the network and index.

Closes #9501

### CODE REVIEW

File: suite-common/wallet-core/src/accounts/accountsActions.ts [hunk 1]
```diff
@@ -29,6 +29,64 @@ const removeAccount = createAction(
     }),
 );
 
+const prepareCreateAccountActionPayload = (
```
Comment (juriczech) [inline suite-common/wallet-core/src/accounts/accountsActions.ts]: Please don't use prepare prefix as it evokes prepareAction  https://redux-toolkit.js.org/api/createAction#using-prepare-callbacks-to-customize-action-contents

**Review by juriczech — CHANGES_REQUESTED** [review]

File: suite-common/wallet-core/src/accounts/accountsActions.ts [hunk 2]
```diff
@@ -29,6 +29,64 @@ const removeAccount = createAction(
...
+    deviceState: string,
+    discoveryItem: DiscoveryItem,
+    accountInfo: AccountInfo,
+    imported?: boolean,
+    accountLabel?: string,
```
Comment (juriczech) [inline suite-common/wallet-core/src/accounts/accountsActions.ts]: please use named params

Comment (PeKne) [reply suite-common/wallet-core/src/accounts/accountsActions.ts]: fixed f3f6acc07d

File: suite-common/wallet-core/src/accounts/accountsActions.ts (hunk 1 above)
Comment (PeKne) [reply suite-common/wallet-core/src/accounts/accountsActions.ts]: fixed f3f6acc07d

Comment (juriczech) [reply suite-common/wallet-core/src/accounts/accountsActions.ts]: Regarding the rename, how about `composeCreateAccountPayload`?

Comment (PeKne) [reply suite-common/wallet-core/src/accounts/accountsActions.ts]: opinionated nit IMO

Comment (juriczech) [reply suite-common/wallet-core/src/accounts/accountsActions.ts]: Yes. That's a problem and not worth discussion / consideration?

Comment (PeKne) [reply suite-common/wallet-core/src/accounts/accountsActions.ts]: I meant by my previous answer, that it is opinionated and the renaming does not change anything. That's all.  But it seems like you really insist, so here is a fixup: 7b18890311. But please, after many previous discussions once again, don't be so pushy on small things as this one.

Comment (PeKne) [reply suite-common/wallet-core/src/accounts/accountsActions.ts]: Fixed in https://github.com/trezor/trezor-suite/pull/9705/commits/7b18890311401c36a31b658d89e5fc2193dbeeb1

**Review by juriczech — APPROVED** [review]

**Pull request merged**